presentation/src/main/java/io/forus/me/android/presentation/view/screens/vouchers/item/VoucherFragment.kt

```diff
@@ -141,19 +145,22 @@ class VoucherFragment : ToolbarLRFragment<VoucherModel, VoucherView,
         rv_transactions.layoutManager = LinearLayoutManager(context)
         rv_transactions.adapter = adapter
 
+
         info_button.setOnClickListener {
-            val url = when {
-                voucher?.product?.id != null ->
-                    "https://" + BuildConfig.PREFIX_URL + "zuidhorn.forus.io/#!/products/${voucher?.product?.id}"
-                voucher?.fundWebShopUrl?.isNotEmpty() == true -> voucher?.fundWebShopUrl
-                else -> "https://www.zuidhorn.nl/kindpakket"
+            if(voucher!!.isProduct) {
+
+                val url: String = if(voucher?.product?.id != null && voucher?.fundWebShopUrl?.isNotEmpty()!!){
+                    voucher?.fundWebShopUrl + "products/" + voucher?.product?.id
+                }else{
+                    "https://kerstpakket.forus.io/"
```

Comment:
I would refer them to fundwebshopUrl first if possible; and if not found to forus.io (not kerstpakket)